domain/src/main/java/org/oppia/domain/question/QuestionProgressController.kt

```diff
@@ -0,0 +1,18 @@
+package org.oppia.domain.question
+
+import org.oppia.app.model.Question
+import javax.inject.Inject
+import javax.inject.Singleton
+
+
+/** Controller for retrieving an exploration. */
```

Comment:
Nit: please update to be question-specific (I suggest using the high-level summary of ``ExplorationProgressController`` for an example, but feel free to leave out the additional bits--I'll fill those in my interface introduction PR).

Reply:
Done